src/emu/debug/debugvw.h

```diff
@@ -266,7 +266,7 @@ class debug_view_expression
 
 	// setters
 	void mark_dirty() { m_dirty = true; }
-	void set_string(const char *string) { m_string.assign(string); m_dirty = true; }
+	void set_string(const std::string &string) { m_string = string; m_dirty = true; }
```

Comment:
This is likely less efficient when the source isn't a persistent `std::string` to begin with.